.github/ISSUE_TEMPLATE/new-feature-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Feature request
+about: Request a new feature for Skin
+title: "[NEW FEATURE]"
+labels: 'type: feature'
+assignees: ''
+
+---
+
+**Describe the feature/module in detail**
+A description of how the feature/module is intended to work.
+
+**Screenshots**
+Add screenshots of the feature/module.
+
+**Figma Link**
+Add a link to the Figma page for the feature/module.
+
+**Prerequisites**
+
+- [ ] Successful design review with Design System Team.
+- [ ] Supports dark mode.
+- [ ] Supports responsive design.
+- [ ] Tokens have been provided (or already exist)
+- [ ] Passed accessibility review
```

Comment:
```suggestion
- [ ] Successful accessibility review with Core A11Y Team.
```

Reply:
I'm not sure why we went back on the wording here, but I want to make it completely unambiguous that every new feature going into ebayui needs core a11y team review.

Reply:
That was @agliga 's suggestion.

Reply:
Just wanted to simplify the wording. 
But we should do it across the board if we do it

Reply:
Let's keep it explicit. Also, we may want a footnote to explain that "successful review" means those teams give their blessing for it to proceed to engineering.

Reply:
Done.